commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala

```diff
@@ -38,6 +39,10 @@ abstract class BlockingUtils {
   def asTask[T](blockingCode: => T): Task[T] =
     Task.eval(blockingCode).executeOn(ioScheduler, forceAsync = true)
 
+  def asIO[A, B](blockingCode: => Either[A, B]): IO[A, B] = IO.deferTotal {
```

Comment:
Can you please debrace this into one line?

Reply:
done